### PR TITLE
Fix portfolio section issues

### PR DESCRIPTION
**Pull request by mamfi-0221**
Update frontend code to pick up changes from [portfolio schema fields](https://github.com/webriq-pagebuilder/sanity-packages/pull/10) and fix layout.

### CODE REVIEW

File: component/sections/portfolio/variant_a.js [hunk 1]
```diff
@@ -1,136 +1,20 @@
 import React from "react";
 import { urlFor } from "../../../lib/sanity";
 
-function VariantA({ /* template,*/ caption, title, images, primaryButton }) {
-  const [category, setCategory] = React.useState("category_2");
+function VariantA({ caption, title, portfolios }) {
+  const [category, setCategory] = React.useState(portfolios?.[0]?.category); //set the first index category as initial value
+  let [viewProjects, setViewPortfolios] = React.useState(8);
```
Comment (webriq) [inline component/sections/portfolio/variant_a.js:6]: What is `8`? For cases like this, it's better if you initialize a variable with the said value `8` and use that variable for better readability. For example, `React.useState(projectsLength)`

Comment (webriq) [reply component/sections/portfolio/variant_a.js:6]: Also, it's better and common practice we make sure that we name `useState` destructured values as first - `name` and 2nd is prefixed with `set` and same name of first. 

Eg:

`const [viewProjects, setViewProjects] = React.useState()`

File: component/sections/portfolio/variant_b.js [hunk 2]
```diff
@@ -1,182 +1,95 @@
...
-                    : primaryButton.externalLink
-                }
+            {portfolios.length === viewProjects || portfolios.length < 6 ? null : (
+              <button 
+                className="hidden md:inline-block py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-gray-50 font-bold leading-loose transition duration-200" 
+                onClick={handleViewMoreProjects}
               >
-                {primaryButton.label}
-              </a>
+                View More Projects
```
Comment (webriq) [inline component/sections/portfolio/variant_b.js:38]: Is this intentional? We've gone from dynamic label to a static text `View More Projects`

**Review by webriq — CHANGES_REQUESTED** [review]

File: component/sections/portfolio/variant_a.js (hunk 1 above)
Comment (mamfi-0221) [reply component/sections/portfolio/variant_a.js:6]: Based on variant design, we may set the number of portfolios to initially display. Here, for variant A I set it to display 8 items on first render. Revised the line of code to use a constant variable for better readability and the function name.

File: component/sections/portfolio/variant_b.js (hunk 2 above)
Comment (mamfi-0221) [reply component/sections/portfolio/variant_b.js:38]: No, I missed to update this one including its corresponding schema field. Revised code including [here](https://github.com/webriq-pagebuilder/sanity-packages/pull/10). Thank you!

Comment (dorelljames) [reply component/sections/portfolio/variant_b.js]: Awesome!

File: component/sections/portfolio/variant_a.js (hunk 1 above)
Comment (dorelljames) [reply component/sections/portfolio/variant_a.js]: Thanks for accepting my proposal for change.

**Review by dorelljames — APPROVED** [review]

**Review by webriq — APPROVED** [review]

**Pull request merged**